types/node-redis-pubsub/node-redis-pubsub-tests.ts

```diff
@@ -0,0 +1,19 @@
+import * as NRP from "node-redis-pubsub";
```

Comment:
Since `NRP` is used as function here, it should be `import NRP = require("node-redis-pubsub")` here